src/api/spec/helpers/webui/buildresult_helper_spec.rb

```diff
@@ -1,100 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Webui::BuildresultHelper do
-  include Webui::WebuiHelper
-  let(:project) { create(:project) }
-  let(:package) { create(:package, project: project) }
-  let(:arch) { 'i586' }
-  let(:repo) { create(:repository, project: project, architectures: [arch]) }
-
-  describe '#arch_repo_table_cell' do
```

Comment:
This helper is still present. We should update these specs instead of removing them.

Reply:
I will update the specs.

Reply:
Updated. Please re-review.